includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -169,6 +172,10 @@ public function register() {
 			}
 		}
 
+		if ( Feature_Flags::enabled( 'rrmModuleV2' ) ) {
+			add_action( 'enqueue_block_editor_assets', $this->get_method_proxy( 'enqueue_block_editor_assets_for_non_sitekit_user' ), 40 );
+		}
```

Comment:
We may as well just move this above into the existing check where we register the blocks, as there's no point in enqueuing them unless the blocks are registered.